pkg/api/controller_test.go

```diff
@@ -1150,6 +1197,133 @@ func TestController_DeleteBranchHandler(t *testing.T) {
 	})
 }
 
+func TestController_IngestRangeHandler(t *testing.T) {
+	const (
+		fromSourceURI           = "https://valid.uri"
+		uriPrefix               = "take/from/here"
+		fromSourceURIWithPrefix = fromSourceURI + "/" + uriPrefix
+		after                   = "some/key/to/start/after"
+		continuationToken       = "opaque"
+		prepend                 = "some/logical/prefix"
+	)
+
+	ctx := context.Background()
+	token := continuationToken
+
+	setup := func(count int, err error) (api.ClientWithResponsesInterface, *testutils.FakeWalker) {
```

Comment:
Can we use some other name than 'err' for the second argument?

Reply:
Done.